lakefs_provider/version.py

```diff
@@ -0,0 +1,23 @@
+"""Report package version"""
+
+import sys
+
+if sys.version_info >= (3, 8):
+    from importlib import metadata
+else:
+    import importlib_metadata as metadata
+
+
+def __get_version(package: str) -> str:
+    try:
+        return metadata.version(package)
+    except metadata.PackageNotFoundError as e:
+        print("No package metadata found")
+        return None
+    except Exception as e:
+        print(f"Get version: {e}")
+        return None
```

Comment:
Can we use the standard logging instead of print?

Reply:
No longer used, we're doing your method of reading version.